src/runtime/contributions-flow-test.js

```diff
@@ -29,7 +33,7 @@ describes.realWin('ContributionsFlow', {}, env => {
   let callbacksMock;
   let pageConfig;
   let port;
-  let messageCallback;
+  const messageMap = {};
```

Comment:
This is a cool pattern, but I worry it leaks state across tests. Could the messageMap be declared with `let` instead, and then cleared `beforeEach` test?

Reply:
good idea. done